### PR TITLE
revert of revert of revert of "First Pass on Docs Redesign"

### PR DESCRIPTION
**Pull request by darzu**
We're going to revert the new docs thing and instead bring features piecemeal because there were too many issues encountered and we think this will be faster.

take header (search, logo placement, hamburger button fixed location)
default closed hamburger
new footer
breadcrumb out of header
collapsed by default menu items w/ chevrons
print icon (other icons) [if easy]
consistent sat & lightness across targets
line above H1
left justified for everything
icons above snippets
sticky-list (p2)
new cards for locations with {picture, header, description} (p3)

### CODE REVIEW

File: theme/themes/pxt/globals/site.variables [hunk 1]
```diff
@@ -38,9 +41,13 @@
 @importGoogleFonts: false;
 
 @docsHeaderFont: "Helvetica Neue", Helvetica, Arial, sans-serif;
+<<<<<<< HEAD
```
Comment (jwunderl) [inline theme/themes/pxt/globals/site.variables]: merge conflict

File: theme/themes/pxt/globals/site.variables [hunk 2]
```diff
@@ -6,9 +6,13 @@
 *******************************/
 
 @primaryColor: @pink;
+<<<<<<< HEAD
 @secondaryColor: rgba(0, 0, 0, 0.55);
 @docsPrimaryColor    : #6C196C;
 @docsAccentColor     : #FC6483;
+=======
```
Comment (jwunderl) [inline theme/themes/pxt/globals/site.variables]: merge conflict, take HEAD (`@secondaryColor: rgba(0, 0, 0, 0.55);`)

File: theme/themes/pxt/globals/site.variables (hunk 1 above)
Comment (darzu) [reply theme/themes/pxt/globals/site.variables]: huh weird I wonder why vscode didn't bubble these

File: theme/themes/pxt/globals/site.variables [hunk 3]
```diff
@@ -6,9 +6,7 @@
 *******************************/
 
 @primaryColor: @pink;
-@secondaryColor: rgba(0, 0, 0, 0.55);
-@docsPrimaryColor    : #6C196C;
-@docsAccentColor     : #FC6483;
+@secondaryColor: @grey;
```
Comment (jwunderl) [inline theme/themes/pxt/globals/site.variables]:
```suggestion
@secondaryColor: rgba(0, 0, 0, 0.55);
```

**Pull request merged**